.github/workflows/operator-integration-test.yml

```diff
@@ -183,18 +206,21 @@ jobs:
 
       - name: Test Annotations
         run: |
-          kubectl apply -f integration-tests/manifests/sample-deployment.yaml
```

Comment:
why did we remove this? I assume this is the deployment we are testing the annotations on?

Reply:
I realized we are wasting resources because I am dynamically creating a sample deployment app for each test in a unique namespace. So this kubectl apply didn't do anything for the test.